daemon/container.go

```diff
@@ -330,5 +330,9 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 	}
 
 	// Now do platform-specific verification
-	return verifyPlatformContainerSettings(daemon, hostConfig, config, update)
+	platformWarns, err := verifyPlatformContainerSettings(daemon, hostConfig, config, update)
+	if hostConfig.NetworkMode.IsHost() && len(hostConfig.PortBindings) > 0 {
```

Comment:
Related to the above; I don't think `host` networking is supported on Windows (although with LCOW .. perhaps it's still relevant here); I'll have to check that one. If not supported on Windows we should probably move it in `verifyPlatformContainerSettings()`

Reply:
ping @johnstep @jhowardmsft could you have a look? ^^

Reply:
@msabansal Probably one for you to answer. PTAL.